<launch>

    <rosparam file="$(find gen_comm_pkg)/config/pin_out.yaml" />
    <rosparam file="$(find gen_comm_pkg)/config/parameters.yaml" />
    
    <node pkg="main_menu" name="main_menu_node" type="main_menu_exec.py" output="screen"/>

    <node pkg="i2c_comm_pkg" name="i2c_comm_node" type="i2c_comm.py" output="screen"/>
    
    <node pkg="spi_comm_pkg" name="spi_comm_node" type="spi_comm.py" output="screen"/>

    <node pkg="accumulator_pkg" name="accumulator_node" type="accumulator.py" output="screen"/>

</launch>
